settings/.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
```

Comment:
I wonder if this would be better as a `pre-push` hook? I often make lots of WIP commits locally, and it can be annoying for the linter to run on those, since they're not intended to pass.

Reply:
It used to support pre-push, but the usage for the new version has changed, and pre-push doesn't seem to work anymore 🤔  I've tried for a while and also checked if others have encountered the same issue, but no good news. Let me do more research to see if I can find a solution.

Reply:
Ok, it's not a huge deal if we keep it `pre-commit`, especially if later on we add an action to do it on push.